src/types/widget.ts

```diff
@@ -0,0 +1,9 @@
+import { NotionConfig } from '../service/notion/config';
+import { FormConfig } from './form';
+import { CollapsedFormConfig } from './collapsedForm';
+
+export type Configuration = {
+  form: FormConfig;
+  collapsedForm: CollapsedFormConfig;
```

Comment:
not the best naming, because it is not a form. I'd suggest `widget: {title, description}`